### ISSUE TITLE
Didn't see the correct folder

### ISSUE THREAD

**Issue by CXY573** [da03/Attention-OCR#57]
![2017-12-16 15-48-20](https://user-images.githubusercontent.com/30207116/34068698-e4942ffc-e27a-11e7-9b89-42e57cd6c5bb.png)

**Comment by CXY573** [comment]
there is only a folder name "incorrect" in the result folder,if there is anyone know what's happen?

**Comment by CXY573** [comment]
My log.txt is follow:

2017-12-16 15:46:09,037 root  INFO     step_time: 0.036033, loss: 3.724525, step perplexity: 41.451547
2017-12-16 15:46:09,052 root  INFO     0.000000 out of 991 correct
2017-12-16 15:46:09,233 root  INFO     step_time: 0.180325, loss: 3.672713, step perplexity: 39.358533
2017-12-16 15:46:09,272 root  INFO     0.000000 out of 992 correct
2017-12-16 15:46:09,307 root  INFO     step_time: 0.034654, loss: 3.741922, step perplexity: 42.178996
2017-12-16 15:46:09,315 root  INFO     0.000000 out of 993 correct
2017-12-16 15:46:09,351 root  INFO     step_time: 0.035934, loss: 3.772001, step perplexity: 43.466967
2017-12-16 15:46:09,361 root  INFO     0.000000 out of 994 correct
2017-12-16 15:46:09,429 root  INFO     step_time: 0.064732, loss: 3.741543, step perplexity: 42.162990
2017-12-16 15:46:09,458 root  INFO     0.000000 out of 995 correct
2017-12-16 15:46:09,498 root  INFO     step_time: 0.038936, loss: 3.721210, step perplexity: 41.314354
2017-12-16 15:46:09,513 root  INFO     0.000000 out of 996 correct
2017-12-16 15:46:09,565 root  INFO     step_time: 0.050532, loss: 3.666165, step perplexity: 39.101668
2017-12-16 15:46:09,596 root  INFO     0.000000 out of 997 correct
2017-12-16 15:46:09,661 root  INFO     step_time: 0.055440, loss: 3.703670, step perplexity: 40.596020
2017-12-16 15:46:09,677 root  INFO     0.000000 out of 998 correct
2017-12-16 15:46:09,711 root  INFO     step_time: 0.033894, loss: 3.758518, step perplexity: 42.884833
2017-12-16 15:46:09,725 root  INFO     0.000000 out of 999 correct
2017-12-16 15:46:09,754 root  INFO     step_time: 0.027922, loss: 3.596800, step perplexity: 36.481293
2017-12-16 15:46:09,768 root  INFO     0.000000 out of 1000 correct
2017-12-16 15:46:09,817 root  INFO     step_time: 0.045393, loss: 3.715956, step perplexity: 41.097876
2017-12-16 15:46:09,834 root  INFO     0.000000 out of 1001 correct
2017-12-16 15:46:09,869 root  INFO     step_time: 0.034558, loss: 3.739633, step perplexity: 42.082537
2017-12-16 15:46:09,879 root  INFO     0.000000 out of 1002 correct
2017-12-16 15:46:09,911 root  INFO     step_time: 0.031293, loss: 3.674526, step perplexity: 39.429952
2017-12-16 15:46:09,923 root  INFO     0.000000 out of 1003 correct
2017-12-16 15:46:09,956 root  INFO     step_time: 0.031391, loss: 3.645304, step perplexity: 38.294411
2017-12-16 15:46:09,964 root  INFO     0.000000 out of 1004 correct
2017-12-16 15:46:10,014 root  INFO     step_time: 0.045758, loss: 3.685593, step perplexity: 39.868763
2017-12-16 15:46:10,026 root  INFO     0.000000 out of 1005 correct
2017-12-16 15:46:10,057 root  INFO     step_time: 0.030331, loss: 3.654885, step perplexity: 38.663058
2017-12-16 15:46:10,070 root  INFO     0.000000 out of 1006 correct
2017-12-16 15:46:10,100 root  INFO     step_time: 0.030195, loss: 3.647512, step perplexity: 38.379049
2017-12-16 15:46:10,110 root  INFO     0.000000 out of 1007 correct
2017-12-16 15:46:10,154 root  INFO     step_time: 0.042073, loss: 3.713620, step perplexity: 41.001973
2017-12-16 15:46:10,175 root  INFO     0.000000 out of 1008 correct
2017-12-16 15:46:10,211 root  INFO     step_time: 0.035314, loss: 3.659956, step perplexity: 38.859632
2017-12-16 15:46:10,218 root  INFO     0.000000 out of 1009 correct
2017-12-16 15:46:10,254 root  INFO     step_time: 0.035393, loss: 3.668833, step perplexity: 39.206117
2017-12-16 15:46:10,267 root  INFO     0.000000 out of 1010 correct
2017-12-16 15:46:10,305 root  INFO     step_time: 0.036534, loss: 3.826706, step perplexity: 45.911055
2017-12-16 15:46:10,319 root  INFO     0.000000 out of 1011 correct
2017-12-16 15:46:10,379 root  INFO     step_time: 0.058488, loss: 3.657184, step perplexity: 38.752050
2017-12-16 15:46:10,392 root  INFO     0.000000 out of 1012 correct
2017-12-16 15:46:10,428 root  INFO     step_time: 0.033963, loss: 3.701324, step perplexity: 40.500882
2017-12-16 15:46:10,442 root  INFO     0.000000 out of 1013 correct
2017-12-16 15:46:10,474 root  INFO     step_time: 0.030381, loss: 3.658771, step perplexity: 38.813604
2017-12-16 15:46:10,486 root  INFO     0.000000 out of 1014 correct
2017-12-16 15:46:10,520 root  INFO     step_time: 0.032638, loss: 3.768634, step perplexity: 43.320852
2017-12-16 15:46:10,528 root  INFO     0.000000 out of 1015 correct

**Comment by CXY573** [comment]
I didn't change anything about the code,so it really puzzle me

**Comment by CXY573** [comment]
I didn't change anything about the code,so it really puzzled me
--

**Comment by CXY573** [comment]
Obviously,the picture in the incorrect folder didn't have any change,there is 30 pictures in the sample folder which is exactlly same
![image_6](https://user-images.githubusercontent.com/30207116/34068735-afa1a436-e27b-11e7-9cb6-881c96ba6e0d.jpg)

**Comment by ashishguptasanu** [comment]
I got the same issue. Please Help

**Comment by XuXuanhui** [comment]
> Obviously

Obviously you need then model  wget http://www.cs.cmu.edu/~yuntiand/model.tgz
tar zxf model.tgz
